dolphinscheduler-dao/src/test/java/org/apache/dolphinscheduler/dao/mapper/ResourceMapperTest.java

```diff
@@ -76,8 +76,10 @@ private Resource insertOne(){
         resource.setDirectory(false);
         resource.setType(ResourceType.FILE);
         resource.setUserId(111);
-        resourceMapper.insert(resource);
-        return resource;
+        if( resourceMapper.insert(resource)>0 ){
+            return resource;
+        }
+        throw new RuntimeException("insert data error");
```

Comment:
If the RuntimeException, how the invokers to handle it ?

Reply:
I also think assert.fail() is better.

Reply:
Thanks for your review, I will finish it later